src/handlers/assign.rs

```diff
@@ -487,10 +487,20 @@ pub(super) async fn handle_command(
                     name.to_string()
                 } else {
                     let teams = crate::team_data::teams(&ctx.github).await?;
-                    // Determine if assignee is a team. If yes, add the corresponding label
-                    // team name here is without prefix 't-' (e.g. 'compiler', 'libs', etc.)
-                    if let Some(team) = teams.teams.get(&name) {
-                        let t_label = format!("t-{}", &team.name);
+                    // remove "t-" or "T-" prefixes before checking if it's a team name
```

Comment:
Can we just trim T- and t- instead of this complicated logic?

Reply:
good point, I should have thought about it.